x-pack/plugins/canvas/common/lib/fonts.js

```diff
@@ -4,53 +4,102 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import { i18n } from '@kbn/i18n';
+
 export const americanTypewriter = {
-  label: 'American Typewriter',
+  label: i18n.translate('xpack.canvas.lib.fonts.americanTypewriterFontLabel', {
```

Comment:
americanTypewriterFontLabel => americanTypewriterLabel. please also remove font in other ids in this file